target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -64,7 +65,9 @@ contract EntropyTest is Test {
         uint64 startSequenceNumber,
         uint64 size
     ) public pure returns (bytes32[] memory hashChain) {
-        bytes32 initialValue = keccak256(abi.encodePacked(startSequenceNumber));
+        bytes32 initialValue = keccak256(
+            abi.encodePacked(provider, startSequenceNumber)
```

Comment:
this was a bug in previous versions where the hash chains for both providers were exactly the same. turns out i wasn't using provider2's hash chain in any of the other tests though.